__tests__/app.test.js

```diff
@@ -92,3 +98,22 @@ describe("GET /api/articles/:article_id", () => {
       });
   });
 });
+
+describe.only("GET /api/articles", () => {
+  test("200 - responds with an array of article objects", () => {
+    return supertest(app).get("/api/articles").expect(200);
+  });
+  test("200 - responds with an array of article objects sorted by date in descending order", () => {
+    return supertest(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body: { articles } }) => {
+        expect(articles.length).toBeGreaterThan(0);
+        //***BELOW**** ITS SAYING TOBESORTED IS NOT A FUNCTION */
+
+        // expect(articles).toBeSortedBy("created_at", {
+        //   descending: true,
+        // });
+      });
```

Comment:
toBeSortedBy is an external package, which means we need to both install it and then require it into the file where we want to use it.


https://www.npmjs.com/package/jest-sorted

Reply:
//cannot add "jest": {"setupFilesAfterEnv": ["jest-sorted"] to package.json without messing up the util-test and jest-extended/all dependency
        //}